github/repos.go

```diff
@@ -327,6 +327,11 @@ type createRepoRequest struct {
 // Note that only a subset of the repo fields are used and repo must
 // not be nil.
 //
+// Also note that this method will return the response without actually
+// waiting for Github to finish creating the repository and letting the
```

Comment:
```suggestion
// waiting for GitHub to finish creating the repository and letting the
```

Reply:
I updated this and took the liberty of updating the same in some other comments as well; hope that's fine.